build.gradle

```diff
@@ -998,6 +998,10 @@ project(':iceberg-spark3') {
     testCompile project(path: ':iceberg-hive-metastore', configuration: 'testArtifacts')
     testCompile project(path: ':iceberg-api', configuration: 'testArtifacts')
     testCompile project(path: ':iceberg-data', configuration: 'testArtifacts')
+    //snappy-java supports Apple M1 cpu at the version of 1.1.8.2 when the tests are executed with JDK 11
```

Comment:
Yes, it can be executed successfully with Java 8 in IntelliJ. Do you think this fix is still necessary in build.gradle?

Reply:
My concern is that it may not be accepted upstream if Java 11 is indeed not supported. I would skip it and ask to use Java 8. You can also try to find the answer to which Java is supported in the documentation to see what the official support is.

Reply:
It supports Java 11 as shown in https://github.com/apache/iceberg. I have also tried to build different branches from Apache repo with Java 11. It succeeds in the branches of 0.12.x, 0.13.x and master, as the version of `snappy-java` has already been upgraded to 1.1.8.2 according to the result of checking dependencies with gradle command.
As our branch of li-0.11.x is forked from the branch of 0.11.x, I also tried to build this branch from Apache repo, however it fails because the dependency of guava-28.0-jre cannot be resolved. I guess the branch of 0.11.x in Apache repo has not been maintained for a long time.

Reply:
Thanks for looking into this! Seems the problem is that we are on an old version of Iceberg. To minimize diversion with Apache Iceberg, I would keep it and get it for free on the upgrade. One of the programs we are running now in the group is to stick to Apache Iceberg and not diverge as much as possible.

Reply:
Sure, this change is removed.